### ISSUE TITLE
What constitute a training dataset?

### ISSUE THREAD

**Issue by sanelec1989** [FarhadZanjani/Histopathology-Stain-Color-Normalization#4]
Hi... Is the training data same as the template data?
To make my question more clear let's assume I have to match Data from Lab A to Lab B, so,
1. Will the training set include data from LabA, LabB, or combination of both
2.  As mentioned the network runs on tiles, therefore will the template consist of all tiles of a particular image or only a subset of those
3. Also at test time how does the cluster to cluster mapping happens between template and test image
(How does the algorithm decide if cluster 0 for A corresponds to cluster 0\1\2\3 of B

**Comment by Nandayang** [comment]
Hi @FarhadZanjani and  @sanelec , have you solved this problem? I'm getting confused when implementing the code. I don't know what is the training set means
